object_database/frontends/database_server.py

```diff
@@ -19,13 +19,17 @@
 import time
 import typed_python
 import object_database.tcp_server as tcp_server
+from object_database.util import sslContextFromCertPathOrNone
+
 from object_database.persistence import InMemoryPersistence, RedisPersistence
 
+
 def main(argv):
     parser = argparse.ArgumentParser("Run an object_database server")
 
     parser.add_argument("host")
     parser.add_argument("port", type=int)
+    parser.add_argument("--ssl-path", default=None, required=False, help="path to (self-signed) SSL certificate")
     parser.add_argument("--redis_port", type=int, default=None)
```

Comment:
Underscores in CLI flag names is an anti-pattern. The python argparse knows to store the value in `redis_port` for the flag `--redis-port`.